.github/workflows/lint.yml

```diff
@@ -41,3 +41,28 @@ jobs:
         with:
           command:      fmt
           args:         --all -- --check
+  check-spellcheck:
+    name:               Check For Spelling and/or Grammar Mistakes
+    runs-on:            ubuntu-latest
+    env:
+      RUST_BACKTRACE:   full
+    steps:
+      - name:           Cancel Previous Runs
+        uses:           styfle/cancel-workflow-action@0.4.1
+        with:
+          access_token: ${{ github.token }}
+      
+      - name:           Checkout sources & submodules
+        uses:           actions/checkout@master
+        with:
+          fetch-depth:  5
+          submodules:   recursive
+      
+      - name:           Add cargo-spellcheck
+        run:            cargo install cargo-spellcheck
+      
+      - name:           Run spellcheck
+        uses:           actions-rs/cargo@master
+        with:
+          command:      spellcheck
+          args:         check  -m 1
```

Comment:
consider adding `-vv` to have some additional output in CI

Reply:
I have looked at the [docs](https://github.com/drahnr/cargo-spellcheck) and there is written:

> To increase verbosity add `-v` (multiple) to increase verbosity.

That is why I added `-v`, correct me please if I am wrong with that one `-v`

Reply:
`-q` nothing
`-` error
`-v` warn
`-vv` debug
and any more than that will probably kill the output buffer

Reply:
Thanks! Oh, my goodness! There is around 400 places to improve:

![image](https://user-images.githubusercontent.com/1605705/115248729-3a12b380-a128-11eb-971d-baa5071bb0db.png)

But as I said `A` I will say `Z`, will deliver it, take few days but will deliver.